src/zcl_abapgit_repo_srv.clas.abap

```diff
@@ -539,9 +540,15 @@ CLASS zcl_abapgit_repo_srv IMPLEMENTATION.
 
     lt_tadir = zcl_abapgit_factory=>get_tadir( )->read( io_repo->get_package( ) ).
 
-    zcl_abapgit_objects=>delete( it_tadir  = lt_tadir
-                                 is_checks = is_checks
-                                 ii_log    = ri_log ).
+    TRY.
+        zcl_abapgit_objects=>delete( it_tadir  = lt_tadir
+                                     is_checks = is_checks
+                                     ii_log    = ri_log ).
+      CATCH zcx_abapgit_exception INTO lx_error.
+        " If uninstall fails, repo needs a refresh to show which objects where deleted and which not
+        io_repo->refresh( ).
+        zcx_abapgit_exception=>raise( lx_error->get_text( ) ).
```

Comment:
```suggestion
        RAISE EXCEPTION lx_error.
```

Reply:
of course! i think i need a vacation from abap ;-)